### PR TITLE
Fix `MESSAGE_USAGE` for editTxn

### LINKED ISSUE #209: BUG: error message related to category for editTxn
Lack spacing
Screenshot 2024-10-29 at 10 08 41 PM

### PR DESCRIPTION
**Pull request by seanlim**
Closes #209 

Follows `edit` command as closely as possible

### CODE REVIEW

File: src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -41,19 +41,26 @@ public class EditCommand extends Command {
...
     public static final String MESSAGE_USAGE =
-            COMMAND_WORD + ": Edit a transaction.\n" + "Parameters: " + " INDEX (must be a positive integer) "
-                    + PREFIX_PHONE + "CONTACT " + PREFIX_AMOUNT
-                    + "AMOUNT " + PREFIX_DESCRIPTION + "DESCRIPTION " + "[" + PREFIX_DATE + "DATE ]" + PREFIX_CATEGORY
-                    + "FOOD\n" + "Example: " + COMMAND_WORD + " " + PREFIX_PHONE + "88888888 " + PREFIX_AMOUNT
-                    + "10.00 " + PREFIX_DESCRIPTION + "Paid John for lunch" + PREFIX_DATE + "23012024 "
-                    + PREFIX_CATEGORY + "FOOD";
+            COMMAND_WORD + ": Edit a transaction.\n"
+                    + "Parameters: INDEX (must be a positive integer) "
+                    + "[" + PREFIX_PHONE + "CONTACT] "
```
Comment (SeeYangZhi) [inline src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: The phone field isn't needed, right?

**Review by SeeYangZhi — CHANGES_REQUESTED** [review]

File: src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java (hunk 1 above)
Comment (seanlim) [reply src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: hmm why not?

Comment (seanlim) [reply src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: when you say "isn't needed" do you mean it's optional? or that `editTxn` should not support editing the person field of a Transaction?

Comment (SeeYangZhi) [reply src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: Yeah I am not sure if  `editTxn` should support editing the person field of a Transaction.

Comment (SeeYangZhi) [reply src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: I didn't realise it was supported.

Comment (seanlim) [reply src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: As it is currently supported, it should be reflected in the UG

Comment (SeeYangZhi) [reply src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java:46]: Agreed

**Review by SeeYangZhi — APPROVED** [review]
LGTM, might create a issue to discuss about the support of modifying person contact in editTxn commamd

**Pull request merged**